end-to-end-test/shared/specUtils.js

```diff
@@ -18,6 +18,11 @@ function waitForCoExpressionTab(timeout) {
     $('//*[@id="coexpressionTabGeneTabs"]').waitForExist(timeout || 20000);
 }
 
+function waitForPatientView(timeout) {
+    $('//*[@id="patientViewPageTabs"]').waitForExist(timeout || 20000);
```

Comment:
what is this selector syntax?

Reply:
If my memory serves me right, all elements anywhere in the hierarchy, that have an attribute id equal to patientViewPageTabs.

Reply:
hm.. is it equivalent to `#patientViewPageTabs` ?

Reply:
If both XPath and CSS selector syntax can be used, then yes, they should be equivalent.

Reply:
maybe we can just use # since its better known and equivalent?

Reply:
@pvannierop ^

Should be fine, right?

Reply:
I have updated the selector to state `#patientViewPageTabs`.

Reply:
can you push the change?

Reply:
Its pushed AFAIK.